oclif/plugin-autocomplete
# feat: add basic autocomplete for powershell [WIP]

## What does this PR do?
Adds very basic autocomplete functionality for powershell.
This will work when hitting `<tab>` right after typing the CLI name but won't work correctly if you hit `<tab>` after having typed in a topic or command.

e.g.
`myCli <tab>` will show you all the commands and nested commands for `myCli`
but `myCli topic <tab>` will not work appropriately.

@W-11791541@
@W-11791492@
Initial efforts for #2 

## Testing instructions
1. Clone this repo and pull down this branch
1. At the root of this repo on your local
   1. Run `yarn install && yarn build`
   1. Using an oclif CLI with [`@oclif/plugin-plugins`](https://github.com/oclif/plugin-plugins/) installed run `<cli> plugins link .`
1. Run `<cli> autocomplete powershell` and follow the instructions that are printed out
1. Type `<cli> <tab>` in your terminal and see autocomplete show up
1. Ensure it works as expected for powershell
